wpimath/src/main/native/include/frc/controller/SimpleMotorFeedforward.h

```diff
@@ -40,7 +39,27 @@ class SimpleMotorFeedforward {
   constexpr SimpleMotorFeedforward(
       units::volt_t kS, units::unit_t<kv_unit> kV,
       units::unit_t<ka_unit> kA = units::unit_t<ka_unit>(0))
-      : kS(kS), kV(kV), kA(kA) {}
+      : kS(kS), kV(kV), kA(kA) {
+    if (kV.value() < 0) {
+      wpi::math::MathSharedStore::ReportError(
+          "kV must be a non-negative number, got {}!", kV.value());
+      kV = units::unit_t<kv_unit>{0};
+      wpi::math::MathSharedStore::ReportWarning("kV defaulted to 0.");
+    }
+    if (kA.value() < 0) {
+      wpi::math::MathSharedStore::ReportError(
+          "kA must be a non-negative number, got {}!", kA.value());
+      kA = units::unit_t<ka_unit>{0};
+      wpi::math::MathSharedStore::ReportWarning("kA defaulted to 0;");
+    }
+  }
+
+  template <typename U, typename = Distance>
+  constexpr SimpleMotorFeedforward(const SimpleMotorFeedforward<U>& other) {
```

Comment:
This can just be:
```cpp
  constexpr SimpleMotorFeedforward(
      const SimpleMotorFeedforward<Distance>& other) = default;
```
Though it can actually be removed entirely because the compiler generates a default copy constructor as long as the copy/move constructors and copy/move assignment operators aren't specified.